src/docfx/build/page/Markup.cs

```diff
@@ -50,13 +42,14 @@ internal static class Markup
 
         public static MarkupResult Result => t_status.Peek().Result;
 
-        public static (MarkdownDocument ast, MarkupResult result) Parse(string content)
+        public static (MarkdownDocument ast, MarkupResult result) Parse(string content, CultureInfo culture)
         {
             try
             {
                 var status = new Status
                 {
                     Result = new MarkupResult(),
+                    Culture = culture,
```

Comment:
It does not affect `Parse`, tokens are only used in the `ToHtml` part.

Reply:
yes, reverting